api/include/opentelemetry/trace/attribute_value.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <cstdint>
+
+#include "opentelemetry/nostd/span.h"
+#include "opentelemetry/nostd/string_view.h"
+#include "opentelemetry/nostd/variant.h"
+#include "opentelemetry/version.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace trace
+{
+using AttributeValue = nostd::variant<bool,
+                                      int,
```

Comment:
Does plain int help in the case where the caller used a `vector<not explicitly int64>`?

Reply:
Yeah, it allows you to pass arrays of ints

Reply:
Nice.